infra/core/storage/storage-account.bicep

```diff
@@ -1,25 +1,32 @@
+metadata description = 'Creates an Azure storage account.'
 param name string
 param location string = resourceGroup().location
 param tags object = {}
 
-@allowed([ 'Hot', 'Cool', 'Premium' ])
+@allowed([
+  'Cool'
+  'Hot'
+  'Premium' ])
 param accessTier string = 'Hot'
-param allowBlobPublicAccess bool = false
+param allowBlobPublicAccess bool = true
```

Comment:
This became true, so you'll need to pass it in as False in main.bicep

Reply:
Done